sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/TimestampFormatter.scala

```diff
@@ -66,6 +66,13 @@ class Iso8601TimestampFormatter(
   }
 }
 
+class FractionTimestampFormatter(timeZone: TimeZone)
```

Comment:
Sorry, last nit. Can you add some docs here?

Reply:
Added